config/loket-consumer/leidinggevenden-dispatching/initial-sync-dispatching.js

```diff
@@ -36,20 +36,24 @@ async function dispatch(lib, data) {
   if (termObjects.length) {
     originalInsertTriples = termObjects.map(o => `${o.subject} ${o.predicate} ${o.object}.`)
 
-    await transformStatements(fetch, originalInsertTriples).then(
-      transformedInsertTriples => {
-        batchedDbUpdate(
-          muAuthSudo.updateSudo,
-          INGEST_GRAPH,
-          transformedInsertTriples,
-          { 'mu-call-scope-id': MU_CALL_SCOPE_ID_INITIAL_SYNC },
-          endpoint,
-          BATCH_SIZE,
-          MAX_DB_RETRY_ATTEMPTS,
-          SLEEP_BETWEEN_BATCHES,
-          SLEEP_TIME_AFTER_FAILED_DB_OPERATION
-        )
-      }
+    let transformedInsertTriples;
+    try {
+      transformedInsertTriples = await transformStatements(fetch, originalInsertTriples);
```

Comment:
This file is fine but I see that the delta sync file for leidinggevenden still contains .then(), it should be replaced by awaits to ensure the order of execution, especially as we do deletes and inserts :smile: